agent/src/main/java/com/criteo/hadoop/garmadon/agent/headers/RessourceManagerHeader.java

```diff
@@ -11,9 +12,9 @@ private RessourceManagerHeader() {
 
     private Header createCachedHeader() {
         return Header.newBuilder()
-                .withHostname(Utils.getHostname())
-                .withUser(Utils.getUser())
-                .withPid(Utils.getPid())
+                .withHostname(HeaderUtils.getHostname())
+                .withUser(HeaderUtils.getUser())
+                .withPid(HeaderUtils.getPid())
```

Comment:
.withMainClass(HeaderUtils.getArrayJavaCommandLine()[0]) ?

Reply:
was not necessary for the purpose of the dashboard but yes I will add it